core/src/main/clojure/xtdb/operator/scan.clj

```diff
@@ -379,7 +378,11 @@
                                                                    :trie-key trie-key}]))))
 
                                   LiveHashTrie$Leaf
-                                  (recur true (conj leaves [:live {:node trie-node, :rel-rdr (.liveRelation live-table-wm)}])))
+                                  (recur true (conj leaves
+                                                    [:live {:rel-rdr
+                                                            (->>
```

Comment:
minor style: I probably wouldn't thread here - the thing you're threading isn't the thing being transformed